central/complianceoperator/v2/scanconfigurations/datastore/datastore_impl_test.go

```diff
@@ -217,6 +217,41 @@ func (s *complianceScanConfigDataStoreTestSuite) TestGetScanConfiguration() {
 	}
 }
 
+func (s *complianceScanConfigDataStoreTestSuite) TestRemoveClusterFromScanConfig() {
```

Comment:
Should we add a test case to ensure we can delete a cluster when it's the only cluster in the scan config?